docs/source/install/installing_mala.rst

```diff
@@ -7,11 +7,16 @@ Prerequisites
 MALA supports any Python version starting from ``3.10.4``. No upper limit on
 Python versions are enforced. The most recent *tested* version is ``3.10.12``.
```

Comment:
I just saw that this is also outdated. [Here](https://github.com/mala-project/mala/issues/559#issuecomment-2231786678) we say we support up to Python 3.11, is this still true?

Reply:
I think python 3.11. might work, but officially, we only support up to 3.10.4 now. Making sure MALA runs with newer python versions is high on my list of priorities, but for now, it has not been really tested.